File: src/main/java/dev/lampirg/consultationappointment/service/teacher/ConsultationMaker.java
package dev.lampirg.consultationappointment.service.teacher;

import dev.lampirg.consultationappointment.data.teacher.DatePeriod;
import dev.lampirg.consultationappointment.data.teacher.Teacher;

public interface ConsultationMaker {
    void createConsultation(Teacher teacher, DatePeriod datePeriod);
    void deleteConsultation(Teacher teacher, DatePeriod datePeriod);
}

File: src/main/kotlin/dev/lampirg/consultationappointment/service/teacher/ConsultationMaker.kt
package dev.lampirg.consultationappointment.service.teacher

import dev.lampirg.consultationappointment.data.teacher.DatePeriod
import dev.lampirg.consultationappointment.data.teacher.Teacher

interface ConsultationMaker {
    fun createConsultation(teacher: Teacher, datePeriod: DatePeriod)
    fun deleteConsultation(teacher: Teacher, datePeriod: DatePeriod)
}